demos/build.gradle

```diff
@@ -1,10 +1,16 @@
-plugins {
-  id 'org.akhikhl.gretty' version '2.0.0' apply false
+buildscript {
```

Comment:
For me, this is supposed to be legacy style. No?

Reply:
Yes... but something odd is happening if I try and do this any other way.  Not sure if I'm screwing up, or if something is wrong with the way the forked version of the plugin is being published.

Reply:
When going back to the 2.0.0 version it "works" (jacoco plugin is failing). But I can't start the app with the new version when doing `../../gradlew appStart`

Reply:
It's not Jacoco thats broken, it's Jacoco trying to interact with the gretty task. Gretty uses internal gradle APIs that changed in 4.6 and had to updated things.  The broken `appStart` is an oversight, need to make sure the submodules can resolve the runner. Fixing.

Reply:
Seems I tripping on https://github.com/akhikhl/gretty/issues/425

Reply:
Replacing the whole `buildscript` with 
```
plugins {
  id 'org.gretty' version '2.1.0' apply false
}
```
seems to work fine.

Including launching the demos with `appStart`.

Reply:
Right it does today: https://plugins.gradle.org/plugin/org.gretty

> Version 2.1.0 (latest)
> Created 20 March 2018.

I'll update accordingly.

Reply:
ah did not pay attention to the dates (PR or plugin publication) :-)

Reply:
héhé... yes... because I tried that earlier. It wasn't working because of a missing jetty 9...